roles/openshift_sanitize_inventory/tasks/main.yml

```diff
@@ -19,10 +25,11 @@
     openshift_release: "{{ openshift_release | string | regex_replace('^v', '') }}"
   when: openshift_release is defined
 
-- name: Ensure a valid deployment type has been given.
-  # this variable is required; complain early and clearly if it is invalid.
-  when: openshift_deployment_type not in known_openshift_deployment_types
+- name: Validate openshift_release looks like a version
+  when: |
+    openshift_release is defined
+    and not openshift_release | match('\d+(\.\d+){1,3}$')
```

Comment:
ah yes, I guess that's more idiomatic.

speaking of which... was there agreement on fail vs assert?